### PR TITLE
Adding engine linux staging tests to run on Ubuntu

### PR DESCRIPTION
**Pull request by yusuf-goog**
Adding linux vm staging tests to run on ubuntu bots as precusor
for move from Debian to Ubuntu.

Bug:102406

*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by yusuf-goog** [comment]
Putting this up for when the task null bug is resolved.

File: .ci.yaml [hunk 1]
```diff
@@ -10,6 +10,30 @@ enabled_branches:
   - flutter-\d+\.\d+-candidate\.\d+
 
 platform_properties:
+  staging_build_linux:
+    properties:
+      ignore_flakiness: "true"
```
Comment (CaseyHillers) [inline .ci.yaml:15]: This isn't needed for repos outside flutter/flutter

File: .ci.yaml [hunk 2]
```diff
@@ -431,3 +455,234 @@ targets:
...
+  - name: Staging_build_linux Android AOT Engine
+    bringup: true
+    presubmit: true
+    recipe: engine/engine
+    properties:
+      ignore_flakiness: "true"
+      add_recipes_cq: "true"
+      build_android_aot: "true"
+      android_sdk_license: \n24333f8a63b6825ea9c5514f83c2829b004d1fee
+      android_sdk_preview_license: \n84831b9409646a918e30573bab4c9c91346d8abd
```
Comment (CaseyHillers) [inline .ci.yaml:468]: This will produce duplicate artifacts, causing race conditions. Is there a way we can disable uploads for these targets?

Comment (yusuf-goog) [reply .ci.yaml:468]: If I run these through LED, they would upload the artifacts as well?

@keyonghan is there a flag to disable the upload?

**Comment by chinmaygarde** [comment]
Is this patch blocked or is progress likely soon?

**Comment by chinmaygarde** [comment]
Closing this as stale. Please reopen once progress can be made.

**Pull request closed without merging**